### DISCUSSION TITLE
Suggestion: Auto-play next file option

### DISCUSSION THREAD

**Discussion by rich123** [PaulWoitaschek/Voice#1305] [Ideas]
Please include the following details in your bug report:

App-Version: 3.6.1 (from F-Droid)
Phone Name: Moto E4
Android Version: 7.1.1

I installed Voice to use it for playing podcasts, primarily due to the built in variable speed playback.

However, when Voice reaches the end of a podcast mp3, it stops (it also often gets hung, but I added an entry to the github issue for that item) and there appears to be no option to tell it to "start playing next file in file list screen sort order.  Having an option to "auto-play next file" would be useful for podcast playback (I can see the value in not having this for actual books, so it should really be a toggle option).

**Comment by ghost** [comment]
https://github.com/PaulWoitaschek/Voice/issues/493

**Comment by thw0rted** [comment]
I would like either a "play next file" option, or the ability to explicitly make a playback queue. I have several short audiobooks that I like to put on to sleep, and in the player I used to use, I'd make a playlist of several books and just let it play.

**Reply by pktiuk** [reply]
Playlist functionality also would be nice

**Comment by thw0rted** [comment]
I'm still having this issue, and at this point it's my only real complaint about the app. Maybe nobody else leaves books running all night?

**Reply by jmesa92** [reply]
I can't always get to my phone while I'm at work, so a play next option would be great
